test.js

```diff
@@ -196,12 +196,13 @@ describe('shopify-token', function () {
         })
         .reply(200, { access_token: token });
 
-      shopifyToken.getAccessToken(hostname, code, function (err, res) {
-        if (err) return done(err);
-
+      shopifyToken.getAccessToken(hostname, code).then(function (res) {
+        console.log('123')
```

Comment:
Remove these `console.log`.
Mocha also works with promises so you can return the promise and remove the `done` callback.